tests/suites/1_stateful/01_load/01_0000_streaming_load.sh

```diff
@@ -13,6 +13,14 @@ if [ ! -f /tmp/ontime.csv ]; then
 fi
 
 
+# do the Data integrity check
+echo "429c47cdd49b7d75eec9b9244c8b1288edd132506203e37d2d1e70e1ac1eccc7 /tmp/ontime.csv" | sha256sum --check > /dev/null 2>&1
+if [ $? -ne 0 ]; then
+	echo "current dataset is not our stateful test dataset"
```

Comment:
How about giving developers more actionable output? Like: 

```
echo "The downloaded dataset </tmp/ontime.csv> has been corrupted, please remove and fetch it again."
```

Reply:
fix it